code/modules/cm_aliens/structures/trap.dm

```diff
@@ -344,6 +344,17 @@
 	..()
 
 /obj/effect/alien/resin/trap/Crossed(atom/A)
+	if((isStructure(A) && istype(A, /obj/structure/bed)) || (isVehicle(A) && !isVehicleMultitile(A)))
+		var/obj/O = A
```

Comment:
https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#dont-use-abbreviations
one letter var needs to be changed